test_assets/Asset_1.json

```diff
@@ -28,7 +28,7 @@
         "description": null,
         "tags": [],
         "test_source": "SMURF",
-        "test_reference": "https://github.com/NCATSTranslator/Feedback/issues/506",
```

Comment:
Is the removal of all these test_references expected?

Reply:
nope, it looks like the spreadsheet column headers changed during the relay.  fixed.